### DISCUSSION TITLE
Is there any solution to GDRR on nextjs 13?

### DISCUSSION THREAD

**Discussion by alextseitlin** [vercel/next.js#46428] [Help]
This is very important. Is there any solution to GDRR on nextjs 13?

**Comment by ciruz** [comment]
GDPR? Sure you can use a libary like [react-cookie-consent](https://www.npmjs.com/package/react-cookie-consent), if the user accepts, you can trigger something, or save the information inside a cookie, and then go on from there. You can access the information if the user has accepted with getCookieConsentValue();. For example, only integrade youtube when the user has accepted it.
